meta-resin-common/recipes-support/resin-mounts/resin-mounts/etc-machine-id.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Machine-id Mount Point
+Requires=mnt-conf.mount
+After=mnt-conf.mount
+Before=systemd-machine-id-commit.service
+
+[Service]
```

Comment:
That would require new mechanism in yocto to inject stuff in the ext partition and also it will be ugly on the yocto side.